xrootd/xrdfs/file.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package xrdfs
+
+import (
+	"go-hep.org/x/hep/xrootd/internal/xrdenc"
+)
+
+// File implements access to a content and meta information of file over XRootD.
+type File interface {
+	Compression() *FileCompression // Compression returns the compression info.
+	Info() *EntryStat              // Info returns the stat info.
```

Comment:
same comment than on `client.file.Info`.
this should probably mention the possible staleness of the retrieved informations.